Dockerfile

```diff
@@ -244,6 +245,19 @@ RUN TS_VERSIONS="1.7.5 2.1.0 2.1.1 2.2.0 2.2.1 2.3.0 2.3.1 2.4.0 2.4.1 2.4.2 2.5
         /build/scripts/install_timescaledb.sh ${pg} ${TS_VERSIONS} || exit 1 ; \
     done
 
+# As we build many Rust Components, against many PostgreSQL versions, we really benefit from
+# a build cache, even if it only exists ephemerally in the container.
+RUN wget "https://github.com/mozilla/sccache/releases/download/v0.2.15/sccache-v0.2.15-x86_64-unknown-linux-musl.tar.gz"
+RUN tar xzfp sccache*.tar.gz
```

Comment:
tar supports streams, so you can do something like  `wget -O - | tar xzfp`

Reply:
Yeah, will do, still playing with the exact bits here.